docs/introduction/diffusion_intro/diffusion_intro.rst

```diff
@@ -0,0 +1,138 @@
+you have to activate the refinement of missing frames, by setting `refine_missing_frames = True` (see section on :ref:`gaussian refinement<kymotracker_gaussian_refinement>`).
+
+
+FAQ on diffusion analysis
+--------------------------
+- **The diffusion coefficients for some of the tracks are negative, what does that mean?**
+    Sometimes a diffusion coefficient of a single track is negative. For example, when your diffusion coefficient is low, you will have some diffusion coefficients above and some below zero. 
+    The negative values do not have a biological meaning by themselves, but the average of all the diffusion coefficients from different tracks should still give a good, positively values estimate of the diffusion 
+    coefficient (provided that you have enough tracks, and that the protein is diffusing freely). The negative data points *should not* be removed from the dataset, otherwise you get a bias in your estimate. 
+
+- **What happens to the diffusion coefficient if a protein bumps into another protein or against other obstacles?**
+    If there are many diffusive proteins binding to the DNA at once and they are not able to pass each other, the proteins will confine the motion, resulting in sub-diffusive or confined diffusive behavior.
+    If the DNA is held between two beads in an optical tweezer experiment, the beads can also confine the motion of a diffusive protein, if the protein diffuses far enough. 
+    If you are interested in free diffusion of a protein, it is best to keep the density of protein binding events low enough, so the proteins don't meet often. 
+    This will help reducing the number binding events, but not when the motion is so wide that the protein ends up on the beads. If you want to exclude the effect of the beads, you can consider cutting the track 
+    to exclude the part where the protein reaches the bead.
+
+- **How do I account for the motion of the DNA molecule itself?**
+    Though the DNA in an optical tweezer experiment is constrained, it still is fluctuating. The motion of the DNA sets a lower limit on what diffusion coefficient can be observed for a protein diffusing 
+    along the DNA. To find out this lower limit, you can use a static molecule as a reference. The motion of the DNA also can lead to a bias in the estimation of the diffusion coefficient :cite:`vestergaard2014optimal`.`
+    (SHOULD WE MENTION THE BIAS HERE, HOW BIG IS THE BIAS?)
+
+- **How do I analyze anomalous diffusion using Pylake?**
+    If a protein is hindered, for example by obstacles, it can not diffuse freely anymore, and the MSD curve will look different. Similarly, a protein that has some motor activity will have a 
+    differently shaped MSD curve. In both cases, we would refer to the motion as 'anomalous diffusion'. At the moment, Pylake does not have any functionality for analyzing anomalous diffusion.
+    Usually, anomalous diffusion is analyzed by looking at the shape of the MSD curve.
```

Comment:
It is actually easy to implement in Pylake

Fitting the MSD plot with a power law D∆t^α results in:
 α > 1 for directed motion,
 α ≤ 1 for constrained diffusion
 α = 1 for pure diffusion

Reply:
It's not quite that simple. We deliberately do not offer that functionality yet since a naive power law fit of the MSDs with an exponent can easily lead to misinterpretation.

It is very easy to get something that looks like anomalous diffusion when it actually isn't (even when simulating pure free diffusion with observation noise). See for example:

Martin, D. S., Forstner, M. B., & Käs, J. A. (2002). Apparent subdiffusion inherent to single particle tracking. Biophysical journal, 83(4), 2109-2117.
Kepten, E., Weron, A., Sikora, G., Burnecki, K., & Garini, Y. (2015). Guidelines for the fitting of anomalous diffusion mean square displacement graphs from single particle tracking experiments. PLoS One, 10(2), e0117722.

Or how poorly the `MSD` actually performs at estimating this exponent:
Muñoz-Gil, G., Volpe, G., Garcia-March, M. A., Aghion, E., Argun, A., Hong, C. B., ... & Manzo, C. (2021). Objective comparison of methods to decode anomalous diffusion. Nature communications, 12(1), 6253.

In addition, the optimal number of points to use in the fitting is not valid when diffusion is anomalous. I would err on the side of caution here, and wait until we have a better picture of what to advise so that people don't get heavily biased estimates.

Reply:
ok, I'll leave out the detailed suggestion and just say that anomalous diffusion is typically determined by looking at the MSD curve.

Reply:
Does it make sense to cite that first paper that you mention, Martin et al? Is that the one you are planning to implement? Do you know single molecule studies where this paper has been used?

Reply:
Yeah, for now that's fine. We should really get some of this anomalous stuff into `pylake`.